rootfs/scheduler/k8s.py

```diff
@@ -401,21 +439,16 @@ def _check_status(self, resp, app_name):
     def _create_rc(self, name, image, command, **kwargs):
         container_fullname = name
         app_name = kwargs.get('aname', {})
-        app_type = name.split('.')[1]
+        app_type = name.split('-')[-1]
         container_name = app_name + '-' + app_type
-        name = name.replace('.', '-').replace('_', '-')
         args = command.split()
-
-        # First ensure that the namespace was created
-        url = self._api("/namespaces/{}", app_name)
-        resp = self.session.get(url)
-        self._check_status(resp, app_name)
         num = kwargs.get('num', {})
         imgurl = self.registry + "/" + image
         TEMPLATE = RCD_TEMPLATE
         shalen = len(image[image.index(":")+5:])
-        if image[image.index(":")+1:image.index(":")+4] == "git" and shalen == 8:
-            imgurl = "http://"+settings.S3EP+"/git/home/"+image+"/slug"
+        git = image[image.index(":")+1:image.index(":")+4]
+        if git == "git" and shalen == 8 and app_type == 'web':
```

Comment:
Nope the logic to figure out if its buildpack app as we are also checking type web. If a buildpack app has type cmd this will fail